packages/glibc.rb

```diff
@@ -585,8 +585,12 @@ def self.check
   end
 
   def self.postinstall
-    @crew_libcvertokens = `#{CREW_LIB_PREFIX}/libc.so.6`.lines.first.chomp.split(/\s/)
-    @crew_libc_version = @crew_libcvertokens[@crew_libcvertokens.find_index('version') + 1].sub!(/[[:punct:]]?$/, '')
+    if File.exist?("#{CREW_LIB_PREFIX}/libc.so.6")
+      @crew_libcvertokens = `#{CREW_LIB_PREFIX}/libc.so.6`.lines.first.chomp.split(/\s/)
+      @crew_libc_version = @crew_libcvertokens[@crew_libcvertokens.find_index('version') + 1].sub!(/[[:punct:]]?$/, '')
+    else
+      @crew_libc_version = LIBC_VERSION
```

Comment:
Do you mean `@crew_libc_version = version`? (This variable is used to represent the version of `glibc` installed by `crew`)
```ruby
puts "Package glibc version is #{@crew_libc_version}."
```

Reply:
I moved the puts line for that so we're not lying about the package version...